receiver/builder.go

```diff
@@ -13,13 +14,22 @@ import (
 	"go.opentelemetry.io/collector/consumer"
 )
 
-// Builder receiver is a helper struct that given a set of Configs and Factories helps with creating receivers.
+var errNilNextConsumer = errors.New("nil next Consumer")
+
+// Builder receiver is a helper struct that given a set of Configs and
+// Factories helps with creating receivers.
+//
+// Deprecated: this builder is being internalized within the service module,
```

Comment:
Can you add the version this is being deprecated on like we did here: https://github.com/open-telemetry/opentelemetry-collector/commit/2b9697fe09616e1175adcea0d1362272af9b4aed#diff-628140520add1c6bc1e1a54def7b2255f03b89938fd72373b8ba848bbc28fbebR27 ?